packages/palette/src/elements/Modal/ModalBase.story.tsx

```diff
@@ -0,0 +1,85 @@
+import { storiesOf } from "@storybook/react"
+import React, { useState } from "react"
+import { Box } from "../Box"
+import { Button } from "../Button"
+import { Input } from "../Input"
+import { Join } from "../Join"
+import { Spacer } from "../Spacer"
+import { Sans } from "../Typography"
+import { ModalBase, ModalBaseProps } from "./ModalBase"
+
+const Example: React.FC<
+  ModalBaseProps & { dialogChildren?: JSX.Element; bodyChildren?: JSX.Element }
+> = ({ bodyChildren, dialogChildren, ...rest } = {}) => {
+  const [open, setOpen] = useState(false)
+
+  const handleClose = () => setOpen(false)
+
+  return (
+    <>
+      <Button variant="secondaryGray" onClick={() => setOpen(true)}>
+        {{ true: "opened", false: "open" }[`${open}`]}
```

Comment:
Mind pushing this into a self-describing variable? Looking closely its clear this tis logging state, but the form might be easier to parse at a glance later if it was labeled `{buttonState}` (or something).